ctrd/client.go

```diff
@@ -2,47 +2,75 @@ package ctrd
 
 import (
 	"context"
+	"fmt"
+	"sync"
 
 	"github.com/containerd/containerd"
 
 	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
 )
 
-const unixSocketPath = "/run/containerd/containerd.sock"
+const (
+	unixSocketPath    = "/run/containerd/containerd.sock"
+	defaultCapacity   = 5
```

Comment:
Maybe `defaultCapacity` is not so readable. Capacity of what? Maybe others have no ideas about that.